WIP NextWebview

idan opened the pull request:
Add NextWebview that can do both panels and sidebars

idan committed:
WIP NextWebview

idan committed:
Add react-icons

idan committed:
Fix extension local access permissions

idan committed:
Remove snowpack config

idan committed:
Remove codicons

idan committed:
WIP Add routing

mattrothenberg commented on src/webviews/src/index.tsx:
```diff
@@ -28,9 +36,30 @@ window.addEventListener('message', e => {
+  let location = useLocation()
+  let navigate = useNavigate()
+  useEffect(() => {
+    navigate(`/${rootEl.dataset.name}`, { replace: true })
```
If I'm understanding the goal here, we could probably use the `MemoryRouter` instead of `BrowserRouter` and "seed" it with the initial route that we want 👇 

```tsx
<MemoryRouter initialEntries={["something"]} initialIndex={0} />
```

idan replied:
Nice, addressed

idan committed:
Use MemoryRouter

idan committed:
Implement multiple view instances

It's still a static class that retains all its instances
but now it knows how to contend with multiple instances

idan committed:
WIP state stuff

idan committed:
Working state management
